pipelines/HSC/ci_hsc.yaml

```diff
@@ -0,0 +1,45 @@
+description: The DRP pipeline specialized for the ci_hsc test dataset.
+instrument: lsst.obs.subaru.HyperSuprimeCam
+imports:
+  # In the long term, we should try to avoid importing anything from pipelines;
+  # better to refactor anything imported into ingredients.
+  location: "$DRP_PIPE_DIR/pipelines/HSC/RC2.yaml"
+  exclude:
+    - fgcm
+    # Don't run jointcal here...
```

Comment:
Why this comment, but not have one for fgcm? Maybe "Don't run jointcal or fgcmcal because we don't have enough data in this dataset."

Reply:
Just history copied over from the originals.  Will fix.